src/components/index.js

```diff
@@ -1,6 +1,6 @@
 export { default as FilterNavigation } from './FilterNavigation';
 export { default as RepeatableField } from './RepeatableField';
-export { default as InstanceFilters } from './InstanceFilters';
-export { default as HoldingFilters } from './HoldingFilters';
-export { default as ItemFilters } from './ItemFilters';
+export { default as InstanceFilters, instanceFilterRenderer } from './InstanceFilters';
+export { default as HoldingFilters, holdingFilterRenderer } from './HoldingFilters';
```

Comment:
`HoldingFilters` should maybe/probably be renamed `HoldingsRecordFilters` for consistency.